### PR TITLE
[Merged by Bors] - chore(Measure.MeasureSpace): clean up uses of `@`

### PR DESCRIPTION
**Pull request by mattrobball**
We eliminate all possible uses of `@`'s either through deletion or replacement with an explicit argument. A comment about a diamond is slightly clarified to point at #10941.

---

- [x] depends on: #10910

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Measure/MeasureSpace.lean [hunk 1]
```diff
@@ -1806,7 +1809,7 @@ theorem limsup_preimage_iterate_ae_eq {f : α → α} (hf : QuasiMeasurePreservi
 #align measure_theory.measure.quasi_measure_preserving.limsup_preimage_iterate_ae_eq MeasureTheory.Measure.QuasiMeasurePreserving.limsup_preimage_iterate_ae_eq
 
 theorem liminf_preimage_iterate_ae_eq {f : α → α} (hf : QuasiMeasurePreserving f μ μ)
-    (hs : f ⁻¹' s =ᵐ[μ] s) : @liminf (Set α) ℕ _ (fun n => (preimage f)^[n] s) atTop =ᵐ[μ] s := by
+    (hs : f ⁻¹' s =ᵐ[μ] s) : liminf (α := Set α) (fun n => (preimage f)^[n] s) atTop =ᵐ[μ] s := by
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/Measure/MeasureSpace.lean:1812]: Should this have a comment too?

File: Mathlib/MeasureTheory/Measure/MeasureSpace.lean [hunk 2]
```diff
@@ -618,7 +618,8 @@ theorem measure_liminf_eq_zero {s : ℕ → Set α} (h : (∑' i, μ (s i)) ≠
 #align measure_theory.measure_liminf_eq_zero MeasureTheory.measure_liminf_eq_zero
 
 theorem limsup_ae_eq_of_forall_ae_eq (s : ℕ → Set α) {t : Set α}
-    (h : ∀ n, s n =ᵐ[μ] t) : @limsup (Set α) ℕ _ s atTop =ᵐ[μ] t := by
+    (h : ∀ n, s n =ᵐ[μ] t) : limsup (α := Set α) s atTop =ᵐ[μ] t := by
+    -- Need to specify `α := Set α` above because of diamond; see gh issue #16932
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/Measure/MeasureSpace.lean:622]: #16932 doesn't exist; is this https://github.com/leanprover-community/mathlib/pull/16932  ?

Comment (mattrobball) [reply Mathlib/MeasureTheory/Measure/MeasureSpace.lean]: Expanded

File: Mathlib/MeasureTheory/Measure/MeasureSpace.lean (hunk 1 above)
Comment (mattrobball) [reply Mathlib/MeasureTheory/Measure/MeasureSpace.lean:1813]: Added

File: Mathlib/MeasureTheory/Measure/MeasureSpace.lean (hunk 2 above)
Comment (eric-wieser) [reply Mathlib/MeasureTheory/Measure/MeasureSpace.lean]: Let's open a new issue for mathlib 4 with a lean 4 mwe, and reference that

**Comment by eric-wieser** [comment]
Can you merge master to shrink the diff?

**Review by eric-wieser — APPROVED** [review]
bors merge

Thanks!

**Pull request closed without merging**